app/controllers/destinations_controller.rb

```diff
@@ -8,6 +8,8 @@ def index
 
   # GET /destinations/1 or /destinations/1.json
   def show
+    @current_weather = Destination.get_weather(@destination.city)
```

Comment:
Here we have only the happy flow when everything is working. If the weather api will throw an error you will get 500 Internal Server Error. Please find a solution treat errors.

Reply:
This will be handled in another branch